components/item/MediaCard.vue

```diff
@@ -59,19 +86,22 @@
       europeanaIdentifier: {
         type: String,
         default: ''
+      },
+      isSinglePlayableMedia: {
+        type: Boolean,
+        default: false
       }
     },
 
     data() {
       return {
-        oEmbedData: {},
-        ratio: 56.25
+        oEmbedData: {}
       };
     },
 
     computed: {
       displayImage() {
-        return (this.imageSrc !== '') && !isRichMedia(this.media);
+        return (has(this.media, 'thumbnails')) && !isRichMedia(this.media);
```

Comment:
I still don't understand what this is aiming to achieve. `this.media.thumbnails` will always be an object here. If this is to serve unit tests, then set it as an object in the unit tests instead.

Reply:
I added this based on the test ( describe('displayImage') ) its items media props include `ebucoreHasMimeType = 'text/plain'` but I suppose there are no actual items with this type? so isRichMedia will suffice for determining if an item is an image